src/NativeMessageHost/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ if (FIREBREATH)
 endif()
 
 include_directories(
+    ${FB_GECKOSDK_SOURCE_DIR}
```

Comment:
Why have you added this include to the NativeMessageHost project? There is no reason that nativemessagehost should know anything about the geckosdk/npapi headers, so most likely either this is indicative of a bug or it's not needed.